spec/spec_helper_acceptance.rb

```diff
@@ -59,6 +59,16 @@
 aIZ8kxsBjLvpi9KQTHi7Wl6Sw3ecoYdKy+2P8S5xOIpWjs8XVmOWf7Tq1+9KPv3z
 HLw/FDCzntkdq3G4em15CdFlO9BTY4HXiHU=
 -----END CERTIFICATE-----" > /tmp/blah.cert'
+
+      #--- START SELINUX WORKAROUND ---
```

Comment:
Confusingly, that is a different Beaker testing project, run by RedHat 😄 https://beaker-project.org/